### ISSUE TITLE
Linking error when using ABC as a static library

### ISSUE THREAD

**Issue by meyerphi** [berkeley-abc/abc#51]
When linking to ABC as a static library in my project, I receive a linking error in the final step (the actual linking command is more complex due to the build system, I have somewhat simplified it):
```
# /usr/bin/c++ -Wall -pedantic -g strix/main.cc.o lib/abc/libabc.a
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtCountOnesInTruth':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:320: undefined reference to `Abc_TtVerifySmallTruth'
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtCountOnesInCofs':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:332: undefined reference to `Abc_TtVerifySmallTruth'
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtScc6':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:384: undefined reference to `shiftFunc'
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtSccInCofs6':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:410: undefined reference to `shiftFunc'
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtCannonVerify':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:1552: undefined reference to `Abc_TtVerifySmallTruth'
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtCanonicizeAda':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:2585: undefined reference to `Abc_TtVerifySmallTruth'
/usr/bin/ld: lib/abc/libabc.a(dauCanon.c.o): in function `Abc_TtCanonicizeCA':
~/dev/strix/lib/abc/src/opt/dau/dauCanon.c:2646: undefined reference to `Abc_TtVerifySmallTruth'
collect2: error: ld returned 1 exit status
```
I am using GCC 9.1.0 for compiling and linking. The issue can be easily fixed by declaring the following two functions as static in addition to inline, as otherwise the compiler may choose to eliminate them without inlining:
https://github.com/berkeley-abc/abc/blob/390adc39caee1cafbeca46a92f73f349ec318e75/src/opt/dau/dauCanon.c#L305 https://github.com/berkeley-abc/abc/blob/390adc39caee1cafbeca46a92f73f349ec318e75/src/opt/dau/dauCanon.c#L372

**Comment by meyerphi** [comment]
It seems that this was already fixed by #33, but then reverted by ee792bddb66eb8345354b08cbb374bcc922804d3.

**Comment by meyerphi** [comment]
Just saw the fix in df2bce1e40bcc14710cc8e019b0ab5a01176c54f, but this is still missing `shiftFunc` which causes an error. I created a pull request #52 which fixes this, if that is merged this issue can be closed.

**Comment by meyerphi** [comment]
Fixed with cdabd42a26b28f19b883682c26df32c53ab31164
